UI/window-basic-main.cpp

```diff
@@ -2471,15 +2449,26 @@ void OBSBasic::ReceivedIntroJson(const QString &text)
 		return;
 	}
 
+#if OBS_RELEASE_CANDIDATE > 0
+	const char *kLastInfoVersion = "InfoLastRCVersion";
+#elif OBS_BETA > 0
+	const char *kLastInfoVersion = "InfoLastBetaVersion";
+#else
+	const char *kLastInfoVersion = "InfoLastVersion";
+#endif
+	const uint64_t currentVersion = (uint64_t)LIBOBS_API_VER << 16ULL |
```

Comment:
This is constexpr in `ParseUpdateManifest` in `UI/update/win-update.cpp`. Should it also be constexpr here?

Reply:
Sure, should work as well.

Reply:
I mostly meant `currentVersion`, but if the others work too, by all means.